codingQuestion.py

```diff
@@ -1 +1,12 @@
-
```

Comment:
The variable name 'L' is not descriptive. Consider using a more meaningful name.